package.nls.json

```diff
@@ -21,6 +21,25 @@
     "AWS.configuration.description.samcli.debug.attach.timeout": "Maximum amount of time to wait for debugger to connect to a SAM Local debug run (in milliseconds)",
     "AWS.configuration.description.samcli.location": "Location of SAM CLI. SAM CLI is used to create, build, package, and deploy Serverless Applications. [Learn More](https://aws.amazon.com/serverless/sam/)",
     "AWS.configuration.description.telemetry": "Enable the AWS Toolkit to send usage data to AWS. `\"Use IDE settings\"` respects the value specified in \"`telemetry.enableTelemetry`\"",
```

Comment:
I'm very interested in feedback on these descriptions !

Reply:
I'm wondering if we should move these out of nls and straight into `package.json`. As they currently stand, they provide no value to someone attempting to implement these types.

Reply:
These are primarily to drive the tooltips - which, I'm assuming, we want to be able to internationalize at some point.